ThirdParty/PSMTabBarControl/source/PSMDarkHighContrastTabStyle.m

```diff
@@ -31,8 +31,4 @@ - (NSColor *)backgroundColorSelected:(BOOL)selected highlightAmount:(CGFloat)hig
   return [NSColor colorWithCalibratedWhite:value alpha:1.00];
 }
 
-- (CGFloat)fontSize {
-  return 12.0;
```

Comment:
The text is intentionally oversize in the high-contrast themes to aid legibility at the expense of beauty.